packages/components/src/components/contract-card/contract-card.scss

```diff
@@ -218,6 +218,7 @@
     }
     &__symbol {
         margin-left: 0.4rem;
+        font-weight: var(--text-weight-bold);
```

Comment:
Please use `Text` component from `components` package for this, i.e., in the component, the text will be wrapped in a `Text` component like:

> <Text weight='bold'> text goes here </Text>